docs/operating-scylla/nodetool-commands/restore.rst

```diff
@@ -26,7 +26,8 @@ Options
 -------
 
 * ``-h <host>`` or ``--host <host>`` - Node hostname or IP address.
-* ``--endpoint`` - ID of the configured object storage endpoint to load SSTables from
+* ``--endpoint`` - Name of the configured object storage endpoint to load SSTables from.
+  This should be configured as per :ref:`the object storage configuration instructions <object-storage-configuration>`.
 * ``--bucket`` - Name of the bucket to load SSTables from
 * ``--snapshot`` - Name of a snapshot to load SSTables from
```

Comment:
I see that in the code it is `prefix` instead of `snapshot` https://github.com/scylladb/scylladb/blob/3e438d23e12693cbf4df4cc0ad0049f7cefdfca7/tools/scylla-nodetool.cc#L1516-L1518

Reply:
created #21135